sftp/pam_config

```diff
@@ -0,0 +1,2 @@
+auth requisite pam_exec.so expose_authtok debug log=/tmp/aa /usr/local/bin/check_mysql_password.sh
```

Comment:
If I understand correctly, any user will be valid as long as the password provided is the same as the NethVoice admin password, right?